src/Route.elm

```diff
@@ -65,13 +64,32 @@ parser =
         -- Textile specific routes
         , Parser.map TextileSimulatorHome
             (Parser.s "textile" </> Parser.s "simulator")
-        , Parser.map TextileSimulator
-            (Parser.s "textile"
-                </> Parser.s "simulator"
-                </> Impact.parseTrigram
-                </> ViewMode.parse
-                </> TextileQuery.parseBase64Query
-            )
+        , parseTextileSimulator
+        ]
+
+
+parseTextileSimulator : Parser (Route -> a) a
+parseTextileSimulator =
+    Parser.oneOf
+        [ -- We keep this parser for backwards compatible reasons: we used to have the choice
+          -- for a view mode between `simple` and `detailed`, but now it's only `simple`,
+          -- and we used to have a functional unit parameter
+          (Parser.s "textile"
+            </> Parser.s "simulator"
+            </> Impact.parseTrigram
+            -- This is the unused "functional unit" parameter
+            </> Parser.string
+            -- This is the unused "viewmode" parameter
+            </> Parser.string
+            </> TextileQuery.parseBase64Query
```

Comment:
Nit: If we want to keep this BC bit, maybe extract this in some `deprecatedTextileRouteParser` fn to easily identify and remove it when the time comes?

I'd go with removing this BC layer entirely, but I'll leave the final decision up to you.

Reply:
Good point on extracting it. I believe it doesn't cost much to keep it for now, this way we won't brake bookmarks. Maybe we'll revisit (and remove) in the future?